Makefile

```diff
@@ -8,7 +8,7 @@ endif
 TARGETS=awesome
 BUILDDIR=build
 
-all: $(TARGETS) ;
+all: check-unit $(TARGETS) ;
```

Comment:
Do you see a way around this?
Should `make install` from the wrapper Makefile get specialized then maybe?